chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/api/NotJoinedRoomApiType.kt

```diff
@@ -2,6 +2,8 @@ package com.pusher.chatkit.rooms.api
 
 import com.pusher.chatkit.CustomData
 
+// import com.pusher.chatkit.CustomData
+
```

Comment:
that commented out import is unnecessary (it indeed exists above)

Reply:
have removed 👍